pkg/kubectl/cmd/util/factory.go

```diff
@@ -1224,3 +1224,20 @@ func (f *Factory) NewBuilder(thirdPartyDiscovery bool) *resource.Builder {
 
 	return resource.NewBuilder(mapper, typer, resource.ClientMapperFunc(f.ClientForMapping), f.Decoder(true))
 }
+
+func (f *Factory) ResourcesWithPodTemplates() []*meta.RESTMapping {
+	RESTMaps := []*meta.RESTMapping{}
+	ResourcesWithTemplates := []string{"Pod", "ReplicationController", "Deployment", "DaemonSet", "Job", "ReplicaSet"}
+	Mapper, _ := f.Object(false)
```

Comment:
use lowercase variable names - uppercase is reserved for public package names (for functions, types, and variables)